### ISSUE TITLE
Project Information is Missing and/or Incorrect

### ISSUE THREAD

**Issue by asapzacy** [mrmaan4u/Sports-ScoreBoard#1]
Some minor project details are missing and/or incorrectly labeled.. let's get started --

  - **app name**: should be`uxscoreboard` (don't know where `Sports-ScoreBoard` came from?)
  - **package.json**-- `name`, `description`, and `author` fields are incorrect and should be updated (see [uxscoreboard/package.json](https://github.com/asapzacy/uxscoreboard/blob/master/package.json) for the correct information)
  - **README.md** -- I currently do not use `https://cbssports.com`. This needs to be removed ASAP as it is potentially illegal to use in production (all other APIs quote "non-commercial use"). Additionally you are missing the API for all MLB scores (https://statsapi.mlb.com). Lastly, you are missing a. screenshot preview image and b. URL to the production app [https://uxscoreboard.com](https://uxscoreboard.com)

Original repo can be found here https://github.com/asapzacy/uxscoreboard if you need any help. Thanks 👍 

p.s. -- are you possibly looking for maintainers? Let me know, I love sports and react and stuff..

**Comment by binn** [comment]
good shit

**Comment by MikeKlebolt** [comment]
+1

**Comment by anzetal** [comment]
+2

**Comment by alexdiment** [comment]
+3

**Comment by DBX12** [comment]
Boo, you stole the code from @asapzacy (I know, I'm stating the obvious)

**Comment by lewis-carson** [comment]
This should be fixed in the next commit, it is a very serious issue and it could be program breaking 

++

**Comment by AyrA** [comment]
The Project from @asapzacy uses MIT license. I think that license allows this kind of behavior

**Comment by DBX12** [comment]
@AyrA Not exactly. Copying the source code is legit unless you remove the LICENSE.txt file which also contains a copyright notice.

> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

See also:
https://opensource.org/licenses/MIT (License text template)
https://github.com/asapzacy/uxscoreboard/blob/master/LICENSE.md (Original license file with copyright notice)

**Comment by Alexandre-Bazeaud** [comment]
+1

**Comment by crystalzac** [comment]
Any update on this? @mrmaan4u
